### PR TITLE
Added code to address https://github.com/folkswithhats/fedy/issues/568

### PR DESCRIPTION
**Pull request by Xmetalfanx**
The GPG Key was not being imported when installing Hangouts/Google Talk

### CODE REVIEW

File: plugins/hangouts.plugin/install.sh [hunk 1]
```diff
@@ -1,3 +1,6 @@
 #!/bin/bash
 
-dnf -y install https://dl.google.com/linux/direct/google-talkplugin_current_$(uname -i).rpm
+wget https://dl.google.com/linux/linux_signing_key.pub
```
Comment (bddenhartog) [inline plugins/hangouts.plugin/install.sh]: Unnecessary line.

File: plugins/hangouts.plugin/install.sh [hunk 2]
```diff
@@ -1,3 +1,6 @@
...
+rpm --import linux_signing_key.pub
```
Comment (bddenhartog) [inline plugins/hangouts.plugin/install.sh]: Please import from the mirror directly.

File: plugins/hangouts.plugin/install.sh [hunk 3]
```diff
@@ -1,3 +1,6 @@
...
+
+dnf -y install https://dl.google.com/linux/direct/google-talkplugin_current_$(uname -i).rpm
```
Comment (bddenhartog) [inline plugins/hangouts.plugin/install.sh]: Missing newline at EOF.

**Comment by bddenhartog** [comment]
Also, please write a better commit message. Something along the lines of `download google signing key before installing hangouts` would be fine.

**Comment by Xmetalfanx** [comment]
Made (hopefully this is what is needed) requested changes, including importing the key directly from the URL.  I WAS wondering about that when I did it the first time, where the key file was "leftover" and that didn't seem right to me.  I was just going by what was on the Google page, and it turns out in "other stuff" I have done, importing the key the way that (I hope) was requested, is actually how I have done it before.    Also I am not sure what happened with the "Signed by: .." part.   I have started getting in the habit of using my GPG key to sign git commits and didn't intend for that to be in the message but rather when someone looks at my commit, there is that green "Verified" box.  For some reason, it didn't even ask me (locally on my system) for my gpg passphrase when making that commit, even though it was suppose to be setup, like in my other local repos.   Sorry about that

**Comment by bddenhartog** [comment]
>  I WAS wondering about that when I did it the first time, where the key file was "leftover" and that didn't seem right to me

You could have always removed the file afterward; but importing the key directly from the web address is simpler for a few reasons (namely: not having to worry about the file, and simpler error handling).

> Also I am not sure what happened with the "Signed by: .." part

The "Signed-off by: <NAME>" line comes from passing `-s` or `--signoff` to `git-commit`. You can [read the documentation](https://git-scm.com/docs/git-commit#git-commit--s) on it for more information.

Like most things, you can edit your `.gitconfig` to force `git-commit` to _always_ do this. You'll find instructions for that [here](https://git-scm.com/docs/git-config#git-config-formatsignOff).

> I have started getting in the habit of using my GPG key to sign git commits

Awesome! I'm a big purporter of encryption, and signing all of your commits to validate authorship.

> ...didn't intend for that to be in the message but rather when someone looks at my commit, there is that green "Verified" box. For some reason, it didn't even ask me (locally on my system) for my gpg passphrase when making that commit, even though it was suppose to be setup, like in my other local repos.

So, there are a few things that need to happen for you to be able to sign commits.

1. You need to have a GPG key (I'm assuming you've already done this, but [here's a great article to read](https://alexcabal.com/creating-the-perfect-gpg-keypair/) for some basic management practices.
2. You need to let git know about the key. You do this by setting the [`user.signingkey`](https://git-scm.com/docs/git-config#git-config-usersigningKey) configuration value.
3. You need to let GitHub know about the gpg key. Unsurprisingly, they have [a blog article](https://help.github.com/articles/adding-a-new-gpg-key-to-your-github-account/) with instructions. You'll want to paste the output of `gpg --export --armor <key identifier>` in the input field.
3. You need to tell Git that you want to sign using your gpg key when making a commit. There are two ways you can do this:
3.1. Via the command line by executing `git commit -S KEYID` or `git commit --gpgsign KEYID`. See the [documentation](https://git-scm.com/docs/git-commit#git-commit--Sltkeyidgt).
3.2. By telling git you always want to sign every commit, by setting the [`commit.gpgsign`](https://git-scm.com/docs/git-config#git-config-commitgpgSign) value in your `.gitconfig`

Hope this helps!

**Pull request merged**